src/helm/benchmark/window_services/window_service_factory.py

```diff
@@ -64,6 +70,8 @@ def get_window_service(model_name: str, service: TokenizerService) -> WindowServ
                 window_service = GPT4WindowService(service)
             if model_name in get_model_names_with_tag(GPT4_32K_CONTEXT_WINDOW_TAG):
                 window_service = GPT432KWindowService(service)
+            if model_name in get_model_names_with_tag(GPT4_TOKENIZER_TAG):
+                window_service = OpenAIChatWindowService(service)
             else:
```

Comment:
Could you make a new `WindowService` for just `gpt-3.5-turbo-031`? It should be exactly the same as `GPT4WindowService` except with a smaller sequence length.

Reply:
Done